src/Resources/config/services/subscribers.xml

```diff
@@ -12,5 +12,12 @@
 
             <tag name="kernel.event_subscriber" />
         </service>
+
+        <service id="BitBag\ShopwareInPostPlugin\Subscriber\CheckoutFinishSubscriber"
+        >
+            <argument type="service" id="shopware.app_system.guzzle" />
```

Comment:
app_system is for Shopware apps. register your own Guzzle instance as a standalone service yourself. An example from the core:

```
        <service id="shopware.captcha.client" class="GuzzleHttp\Client"/>
```

Reply:
Decided to use WebClient instead of Guzzle